starter_code/script.js

```diff
@@ -1,9 +1,133 @@
-window.onload = function() {
-  document.getElementById("start-button").onclick = function() {
+var car = {
+  x: 600/2-35,
+  y: 850
+}
+
```

Comment:
You could initiate your obstacles here

`
var obstacles_list = []
var counter_for_obstacles = 0
`